src/components/upload/UploadArea.tsx

```
@@ -0,0 +1,101 @@
+          accept=".md,.mdx"
+          onChange={handleFileSelect}
+          className="hidden"
+          aria-label="Upload documentation file"
+        />
+        <div className="w-16 h-16 mx-auto mb-4 bg-dark-800 rounded-full flex items-center justify-center">
+          <svg
+            className="w-8 h-8 text-teal-400"
+            fill="none"
+            viewBox="0 0 24 24"
+            stroke="currentColor"
+          >
```

Biome (2.3.14)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)